### PR TITLE
 Update Xiaomi devices for Configuration variables 

### PR DESCRIPTION
**Pull request by Mryck**
**Description:**

Update Xiaomi devices for Configuration variables 
Related to #6385

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Review by frenck — DISMISSED** [review]

File: source/_components/camera.xiaomi.markdown [hunk 1]
```diff
@@ -53,18 +53,42 @@ camera:
...
+host:
+  description: The IP address or hostname of the camera.
+  required: true
+  type: string
+model:
+  description: The model of Xiaomi Camera, currently supporting yi and xiaofang.
+  required: true
+  type: string
+password:
+  description:The password to the FTP server on the camera (from above), can be any string as the current firmware doesn't allow setting ftp passwords.
```
Comment (WofWca) [inline source/_components/camera.xiaomi.markdown]: Space after `:` is needed.

File: source/_components/camera.xiaomi.markdown [hunk 2]
```diff
@@ -53,18 +53,42 @@ camera:
...
+  required: true
+  type: string
+path:
+  description: The path to the raw MP4 files.
+  required: false
+  default: /media/mmcblk0p1/record
+  type: string
+username:
+  description:The user that can access the FTP server.
```
Comment (WofWca) [inline source/_components/camera.xiaomi.markdown]: Same here

**Comment by frenck** [comment]
Finally! Thanks, @Mryck for this PR and sorry for the fiddling, I was going blind 🙄 

@WofWca Thank you for showing me the way... I was lost. 👍 

Happy Hacktoberfest 🎉 

../Frenck

**Review by frenck — APPROVED** [review]

**Comment by Mryck** [comment]
@frenck No problem thanks for your help !

**Pull request merged**